bofire/data_models/surrogates/quadratic.py

```diff
@@ -0,0 +1,21 @@
+from typing import Literal
+
+from pydantic import Field
+
+from bofire.data_models.kernels.api import (
+    PolynomialKernel,
+)
+from bofire.data_models.priors.api import BOTORCH_NOISE_PRIOR, AnyPrior
+
+# from bofire.data_models.strategies.api import FactorialStrategy
+from bofire.data_models.surrogates.botorch import BotorchSurrogate
+from bofire.data_models.surrogates.scaler import ScalerEnum
+from bofire.data_models.surrogates.trainable import TrainableSurrogate
+
+
+class QuadraticSurrogate(BotorchSurrogate, TrainableSurrogate):
```

Comment:
Rather `PolynomialSurrogate`?

Reply:
This specific one has power=2. So its quadratic.